rbc/remotejit.py

```diff
@@ -723,8 +723,9 @@ def __call__(self, *signatures, **options):
         ------------------
         local : bool
         devices : list
-          Specify device names for the given set of signatures.
-        templates : dict
+          Specify device names for the given set of signatures. Possible
+          values are 'cpu', 'gpu'.
+        templates : dict(str, list(str)) or list(str)
```

Comment:
`list(str)` does not seem to be possible, hence:
```suggestion
        templates : dict(str, list(str))
```

Reply:
Actually it is when you do `T=['...']`. `T` is getting caught by `templates`.

Reply:
Wait no. My mistake indeed!

Reply:
Ok I checked and if you do:

`ljit('T(T)', T=['i8', 'i4'])`

then `templates={T=['i8', 'i4']}`